test/remote/gateways/remote_wompi_test.rb

```diff
@@ -34,6 +34,13 @@ def test_successful_purchase_without_cvv
     assert_success response
   end
 
+  def test_successful_purchase_with_tip_in_cents
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(tip_in_cents: 300))
+    assert_success response
+    # response_data = response.params['data']
+    # assert_equal response_data.dig('tip_in_cents'), 300
+  end
```

Comment:
this test feels kind of lame and anti-climactic, left the commented portions so it would be easy to uncomment and run if desired. It will cause test to fail because it comes back as null in the response which is a bit mystifying. I tried playing around with putting the `add_tip_in_cents` method within the `add_invoice` method to exactly mimic their doc example attached in Jira, but it didn't make a difference with it coming back in the response as a non null value. Thought it best to keep 'em separated. If these ramblings don't make sense, I'll be happy to clarify!

Reply:
Curious! I wonder if we can confirm with Wompi that this value should be echoed back in the response 🤔

Reply:
They confirmed it's expected to be null in sandbox but will show amount in production. Yay.